api/v1alpha1/rule_types_test.go

```diff
@@ -58,6 +58,14 @@ var (
           ]
         }
       }
+    ],
```

Comment:
Again this part would benefit greatly from using https://pkg.go.dev/embed

Reply:
Also changed the integration tests to use embed.